### PR TITLE
feat: Global context shim

### PR DESCRIPTION
**Pull request by ling1726**
Adds a shim for `createContext` that will register the context to the
global scope, and reuse a globally scoped context if it already exists

### CODE REVIEW

**Comment by layershifter** [comment]
Would you mind to add bundle size fixtures?

File: packages/react-components/global-context/bundle-size/createContextSelector.fixture.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { createContextSelector } from '@fluentui/global-context';
+console.log(createContextSelector);
+
+export default {
+  name: 'createContextSelector implementations',
```
Comment (layershifter) [inline packages/react-components/global-context/bundle-size/createContextSelector.fixture.js:5]:
```suggestion
  name: 'createContextSelector',
```

File: packages/react-components/global-context/package.json [hunk 2]
```diff
@@ -30,6 +32,7 @@
     "@fluentui/scripts": "^1.0.0"
   },
   "dependencies": {
+    "@fluentui/react-context-selector": "9.0.0-rc.10",
     "@fluentui/react-theme": "9.0.0-rc.9",
```
Comment (layershifter) [inline packages/react-components/global-context/package.json:36]:
```suggestion
```

Please also remove other unrelated dependencies, too.

Comment (ling1726) [reply packages/react-components/global-context/package.json]: ah good catch, I did a bad merge commit

**Pull request merged**